v0.7/speech_recognition/rnnt/accuracy_eval.py

```diff
@@ -6,9 +6,9 @@
 import sys
 import os
 
-from QSL import AudioQSL
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "pytorch"))
```

Comment:
@galv Could you take a look at this change and see if it makes sense? The context is: MLPerf Inference rules allow SUT to output in pre-qualified datatypes (such as INT8, INT16, INT32, INT64, etc.). Also, in this context, the output is ASCII characters, so we probably want to support other dtypes than INT64. Thanks